lib/sdf-server/src/extract/request.rs

```diff
@@ -59,12 +59,21 @@ impl FromRequestParts<AppState> for ValidatedToken {
             return Ok(Self(claims.clone()));
         }
 
-        let RawAccessToken(raw_token) = parts.extract_with_state(state).await?;
+        let RawAccessToken(raw_token) = parts.extract().await?;
 
         let jwt_public_signing_key = state.jwt_public_signing_key_chain().clone();
         let token = validate_raw_token(jwt_public_signing_key, raw_token)
             .await
             .map_err(unauthorized_error)?;
+
+        // If it has role: Automation, check with auth API if the token has been revoked
```

Comment:
This is the new code that runs everywhere. It's scoped specifically to avoid impact to web ui token usage (which can't be revoked anyway).